apps/src/p5lab/spritelab/poembot/commands/backgroundEffects.js

```diff
@@ -8,6 +8,21 @@ export const commands = {
       this.p5.background(color);
     };
   },
+
+  setBackgroundImageAs(imageName) {
+    if (
+      this.p5._predefinedSpriteAnimations &&
+      this.p5._predefinedSpriteAnimations[imageName]
+    ) {
+      let backgroundImage = this.p5._predefinedSpriteAnimations[imageName];
```

Comment:
nit: we can simplify this with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

```javascript
const backgroundImage = this.p5._predefinedSpriteAnimations?.[imageName];
if (backgroundImage) {
  // set backgroundImage
}
```